### ISSUE TITLE
Adjust the volume of audio notifications

### ISSUE THREAD

**Issue by GentlemanHal** [build-canaries/nevergreen#343]
## Feature Request

**What would you like to be able to do?**
Adjust the volume of audio notifications.

**What problem would it solve?**
This will allow people to manually adjust the volume relative to other media that might be playing on their machines.

We currently play all audio notifications at the default volume, not sure if this is 100% or not.
This should be treated as a personal settings, that can't be overridden by imports.
This could be configured on a per notification basis, but that probably isn't required and a global setting on the Notifications page should be enough.
